front/src/Phaser/Entity/body_character.ts

```diff
@@ -310,3 +311,29 @@ export const loadAllLayers = (load: LoaderPlugin) => {
         }
     }
 }
+
+export const OBJECTS: Array<PlayerResourceDescriptionInterface> = [
+    {name:'report_flag', img:'resources/objects/report_flag.png'},
+    {name:'layout_modes', img:'resources/objects/layout_modes.png'},
+    {name:'teleportation', img:'resources/objects/teleportation.png'},
+];
```

Comment:
The 'report_flag' should not be loaded, right? Since it is now included in HTML rather that in Phaser